smoke-test/src/test/java/org/opennms/smoketest/TopologyIT.java

```diff
@@ -82,24 +82,15 @@
      * Represents a vertex that is focused using
      * criteria listed bellow the search bar.
      */
+    @Data
```

Comment:
Here it gets interesting: '@Data' generates Getters, Setters, equals() and hashCode() methods,  '@NonNull' creates a null check.
42 lines of code become 4 lines :-)

Read more here:
- https://projectlombok.org/features/Data
- https://projectlombok.org/features/NonNull